src/components/Layout/Page.module.css

```diff
@@ -0,0 +1,39 @@
+.container {
+  background-color: var(--color-gray-lightest);
+}
+
+.pageMain {
+  max-width: 100vw;
+  grid-row: auto !important;
+  grid-column: 5 / 14 !important;
+  margin-top: 6rem !important;
+  margin-left: 50px !important;
+  padding-bottom: 12rem;
+}
```

Comment:
This isn't the best way to position this div. Consider using `flex` as I suggested and it should improve readability and reduce the dependency of specific breakpoints. This should be enough:

```
.pageMain {
    grid-row: auto !important;
    margin-top: 6rem !important;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.mainContent {
    max-width: 540px;
}
```

Reply:
Finally saw what you meant, mate. Fixed this.